app/enterprise/changelog.md

```diff
@@ -4,6 +4,195 @@ no_search: true
+  [#7463](https://github.com/Kong/kong/pull/7463)
+- [Prometheus](/hub/kong-inc/prometheus) (`prometheus`) 
+  The new `data_plane_cluster_cert_expiry_timestamp` exposes DP cert expiry timestamp.
+  [#7800](https://github.com/Kong/kong/pull/7800).
+- [GRPC-Gateway](/hub/kong-inc/grpc-gateway) (grpc-gateway)
+  - The plugin can decode entities of type `.google.protobuf.Timestamp`
+    [#7538](https://github.com/Kong/kong/pull/7538)
+  - Added support for structured URI arguments
+    [#7564](https://github.com/Kong/kong/pull/7564)
+- [Request Termination](/hub/kong-inc/request-termination) (`request-termination`)
+  - New `trigger` config option, which makes the plugin activate for any requests with a header or query parameter
+    named like the trigger
+    [#6744](https://github.com/Kong/kong/pull/6744).
+  - The `request-echo` config option. If not set, the plugin activates on all requests.
+    If set, then the plugin only activates for any requests with a header or query parameter
+    named like the trigger
+    [#6744](https://github.com/Kong/kong/pull/6744).
+
+### Dependencies
+- Bumped `openresty` from 1.19.3.2 to [1.19.9.1](https://openresty.org/en/changelog-1019009.html)
+  [#7430](https://github.com/Kong/kong/pull/7727)
+- Bumped `openssl` from `1.1.1k` to `1.1.1l`
+  [7767](https://github.com/Kong/kong/pull/7767)
+- Bumped `lua-resty-http` from 0.15 to 0.16.1
+  [#7797](https://github.com/kong/kong/pull/7797)
+- Bumped `Penlight` to 1.11.0
+  [#7736](https://github.com/Kong/kong/pull/7736)
+- Bumped `lua-resty-http` from 0.15 to 0.16.1
+  [#7797](https://github.com/kong/kong/pull/7797)
+- Bumped `lua-protobuf` from 0.3.2 to 0.3.3
+  [#7656](https://github.com/Kong/kong/pull/7656)
+- Bumped `lua-resty-openssl` from 0.7.3 to 0.7.4
+  [#7657](https://github.com/Kong/kong/pull/7657)
+- Bumped `lua-resty-acme` from 0.6 to 0.7.1
+  [#7658](https://github.com/Kong/kong/pull/7658)
+- Bumped `grpcurl` from 1.8.1 to 1.8.2
+  [#7659](https://github.com/Kong/kong/pull/7659)
+- Bumped `luasec` from 1.0.1 to 1.0.2
+  [#7750](https://github.com/Kong/kong/pull/7750)
+- Bumped `lua-resty-ipmatcher` to 0.6.1
+  [#7703](https://github.com/Kong/kong/pull/7703)
+
+All Kong Gateway OSS plugins will be moved from individual repositories and centralized
+into the main Kong Gateway (OSS) repository. We are making a gradual transition. On this
+release:
+
+- Moved AWS-Lambda inside the Kong repo
+  [#7464](https://github.com/Kong/kong/pull/7464).
+- Moved ACME inside the Kong repo
+  [#7464](https://github.com/Kong/kong/pull/7464).
+- Moved Prometheus inside the Kong repo
+  [#7666](https://github.com/Kong/kong/pull/7666).
+- Moved Session inside the Kong repo
+  [#7738](https://github.com/Kong/kong/pull/7738).
+- Moved GRPC-web inside the Kong repo
+  [#7782](https://github.com/Kong/kong/pull/7782).
+- Moved Serverless functions inside the Kong repo
+  [#7792](https://github.com/Kong/kong/pull/7792).
```

Comment:
I realize this is relevant for the OSS build, but it isn't for the EE build and we should remove the whole section in this changelog